front/app/components/ProjectCard/LargeProjectCard.tsx

```diff
@@ -0,0 +1,333 @@
+  width: 100%;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  padding-top: 20px;
+  max-width: 400px;
+  justify-content: center;
+`;
+
+const ProjectDescription = styled.div`
+  color: ${colors.textSecondary};
+  font-size: ${fontSizes.base}px;
+  line-height: normal;
+  font-weight: 300;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  word-break: break-word;
+  margin-top: 15px;
+
+  ${isRtl`
+    text-align: right;
+  `}
+`;
+
+const ContentFooter = styled.div`
+  flex-shrink: 0;
+  flex-grow: 0;
+  flex-basis: 45px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 8px;
+
+  &.hidden {
+    border: none;
+    margin-top: 0px;
+  }
+`;
+
+export interface InputProps {
+  projectId: string;
+  hideDescriptionPreview?: boolean;
+  className?: string;
+  showFollowButton?: boolean;
+}
+
+const LargeProjectCard = memo<InputProps>(
+  ({ projectId, hideDescriptionPreview, className, showFollowButton }) => {
+    const { data: project } = useProjectById(projectId);
+    const { data: projectImage } = useProjectImage({
+      projectId,
+      imageId: project?.data.relationships.project_images?.data[0]?.id,
+    });
+    const currentPhaseId =
+      project?.data?.relationships?.current_phase?.data?.id ?? null;
+    const { data: phase } = usePhase(currentPhaseId);
+    const fetchPhases = project && !currentPhaseId;
+    const { data: phases } = usePhases(
+      fetchPhases ? project.data.id : undefined
+    );
+    const theme = useTheme();
+
+    if (project) {
+      const imageUrl = !projectImage
+        ? null
+        : projectImage.data.attributes.versions?.large;
+      const projectUrl: RouteType = getProjectUrl(project.data);
+      const isArchived =
+        project.data.attributes.publication_status === 'archived';
+      const hasAvatars =
+        project.data.relationships.avatars &&
+        project.data.relationships.avatars.data &&
+        project.data.relationships.avatars.data.length > 0;
+      const showFooter = hasAvatars;
+      const avatarIds =
+        project.data.relationships.avatars &&
+        project.data.relationships.avatars.data
+          ? project.data.relationships.avatars.data.map((avatar) => avatar.id)
+          : [];
+      const inputTerm = getInputTerm(phases?.data, phase?.data);
+
+      return (
+        <Container
+          className={[
+            className || '',
+            'e2e-project-card',
+            'e2e-admin-publication-card',
+            isArchived ? 'archived' : '',
+            hideDescriptionPreview ? 'hideDescriptionPreview' : '',
+          ]
+            .filter((item) => item)
+            .join(' ')}
+          to={projectUrl}
+          scrollToTop
+          onClick={() => {
+            handleProjectCardOnClick(project.data.id);
+          }}
+        >
+          <ScreenReaderContent project={project} />
+
+          <ProjectImageContainer>
+            {imageUrl ? (
```

Comment:
This line 231 - 248 is exactly repeated in the medium card and in the small card (at 245 - 262). Should we extract this to a separate file and reuse it?